internal/store/atlas/backup.go

```diff
@@ -60,3 +66,34 @@ func (s *Store) EnableEncryptionAtRest(projectID string) (*atlasv2.DataProtectio
 	}
 	return result, nil
 }
+
+func (s *Store) CreatePolicyItem(projectID string, policyItem *atlasv2.DiskBackupApiPolicyItem) (*atlasv2.DataProtectionSettings, error) {
+	compliancePolicy, _, err := s.clientv2.CloudBackupsApi.GetDataProtectionSettings(s.ctx, projectID).Execute()
+	if err != nil {
+		return nil, fmt.Errorf("couldn't create policy item: %w", err)
+	}
+
+	err = addItem(compliancePolicy, policyItem)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't create policy item: %w", err)
+	}
```

Comment:
what's the advantage of the duplicate error wrapping here?

Reply:
I'm not sure what you mean exactly, could you clarify please?

Reply:
From my perspective, a few advantages. I find decorated error messages to be very consistent in giving enough context to fix a problem, as a user and a developer. Of course some of the wraps might be basically same things but with different words, but I'd prefer that over too little context.

Also ties in a bit with go proverbs
> Don't just check errors, handle them gracefully.

Reply:
still doesn't answer

advantage of doing fmt.Errorf("couldn't create policy item: %w", err) twice for different cases, or doing

```
_, err := CreatePolicyItem()
fmt.Errorf("couldn't create policy item: %w", err)
```
once outside the method

Which goes with 
>[Errors are values.](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=16m13s)

Reply:
Agreed, removed wrapping here.